src/main/java/com/websafe/ccmsbe/entity/PrivacyRegulation.java

```diff
@@ -24,7 +35,8 @@ public class PrivacyRegulation {
 
     @ManyToMany(
             mappedBy = "privacyRegulations"
-    )
+    )@JsonBackReference
+    @Column(name= "websites")
```

Comment:
Keep each annotation in a new line.

```suggestion
    )
    @JsonBackReference
    @Column(name= "websites")
```